pkg/client/query/accquerier.go

```diff
@@ -4,18 +4,17 @@ import (
 	"context"
 
 	"cosmossdk.io/depinject"
-	cosmosclient "github.com/cosmos/cosmos-sdk/client"
 	accounttypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	grpc "github.com/cosmos/gogoproto/grpc"
 
 	"github.com/pokt-network/poktroll/pkg/client"
-	"github.com/pokt-network/poktroll/pkg/client/query/types"
 )
 
 // accQuerier is a wrapper around the accounttypes.QueryClient that enables the
 // querying of on-chain account information through a single exposed method
 // which returns an accounttypes.AccountI interface
 type accQuerier struct {
-	clientCtx      types.Context
+	clientCtx      grpc.ClientConn
```

Comment:
1. This is what makes us non-dependent from the `cosomsclient.Context` and thus `cobra.Command` on our query clients. And since they only need a `grpc.ClientConn` (which `cosomosclient.Context` implements), we then reduce our dependency count.
2. Yes